<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget
{
background-image: url(D:\qt_images/Fondo-Login.png);
background-repeat:no-repeat;
}

QFrame#Logologin
{
background-image: url(D:\qt_images/Logo-Login.png);
background-repeat:no-repeat;
}

QPushButton#Enviar
{
background-image: url(D:\qt_images/btn-Login.png);
background-repeat:no-repeat;
background-color: transparent;
font-size:14pt; 
font-weight:550;
}

QPushButton#Enviar:hover
{
border-radius:3px;
background-color: transparent;
}

QCheckBox
{
image: url(D:\qt_images/toggleoff.png)
}

QCheckBox#checkBox:indicator
{
width:120px;
height:100px;
image: none;

}
QCheckBox:checked
{
image: url(D:\qt_images/toggleon.png);
}

QCheckBox:disabled
{
image: url(D:\qt_images/toggleoff.png);
}


QLineEdit
{
background-color: rgb(212, 212, 212);
border-radius:13px;
font-size:14pt;
padding-left:20px;
padding-rigth:10px;
}

QLabel
{
color: rgb(0, 0, 0);
	background-color: qlineargradient(spread:pad, x1:0.192, y1:0.528818, x2:0.699, y2:0.551136, stop:0.474576 rgba(218, 217, 216, 255), stop:1 rgba(234, 234, 234, 255));
border-radius:7px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Logologin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Enviar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pasword">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>229</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pasword2">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>349</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Pasword&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>660</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
